### PR TITLE
Add ERC: Collateralized NFT

### PR DESCRIPTION
**Pull request by harpocrates555**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: collateralizedNFT.md
Comment (abcoathup) [inline collateralizedNFT.md:1]:
```suggestion
---
eip: 7595
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Comment (harpocrates555) [reply collateralizedNFT.md:1]: Sounds good!

Should we move the file to the ERC folder or just leave it in the current location (root directory)?

Comment (abcoathup) [reply collateralizedNFT.md:1]: Needs to be in the ERCs folder.

**Comment by harpocrates555** [comment]
@abcoathup , just following up. Can you please let us know what are the next steps?

**Comment by abcoathup** [comment]
> what are the next steps?

You first need to fix the issues reported: https://github.com/ethereum/ERCs/pull/202/files

Then wait for an ERC editor to review. 
Make any required changes so the PR can be merged.

**Comment by harpocrates555** [comment]
@abcoathup all errors are corrected except external URLs.

Based on this source https://ethereum.github.io/eipw/markdown-rel-links/, external URLs are prohibited.

We have included 3 external URLs with practical implementations and tests of our proposal. What should we do?

**Comment by abcoathup** [comment]
> @abcoathup all errors are corrected except external URLs.
> 
> Based on this source https://ethereum.github.io/eipw/markdown-rel-links/, external URLs are prohibited.
> 
> We have included 3 external URLs with practical implementations and tests of our proposal. What should we do?

Remove the external links, they are not currently allowed. 

Please note ERCs are for standards and not for project promotion. So remove references to your project. 

Once all validator errors have been resolved you will need to wait for an ERC editor.

**Comment by harpocrates555** [comment]
Hi @abcoathup, please let us know what are the next steps?

File: ERCS/erc-7595.md
Comment (xinbenlv) [inline ERCS/erc-7595.md:16]: Please don't forget to update all EIP-721, EIP-20 to ERC-721, ERC-20 etc

File: ERCS/erc-7595.md
Comment (xinbenlv) [inline ERCS/erc-7595.md:20]: The motivation shall focus on a problem to solve, e.g. "There is a use case to do blah blah blah" instead of "our project ABC solves this use case blah blah blah". As project come and go, the standard is meant to stay.

File: ERCS/erc-7595.md
Comment (xinbenlv) [inline ERCS/erc-7595.md:22]:
```suggestion
This ERC addresses various challenges encountered by the crypto community with [ERC-721](./erc-721.md) based collections and assets. This ERC brings forth advantages such as sustainable NFT royalties supported by tangible assets, an on-chain verifiable floor price, and the introduction of additional monetization avenues for NFT collection creators.
```

Comment (xinbenlv) [reply ERCS/erc-7595.md:22]: As we moved the repository, we start to change all files to `erc-`. A kind reminder to update them

File: ERCS/erc-7595.md
Comment (xinbenlv) [inline ERCS/erc-7595.md:65]: How will ERC721 OnReceive behave for this address in this case? Is that behavior defined?

[onERC721Received](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721#IERC721Receiver-onERC721Received-address-address-uint256-bytes-)

File: ERCS/erc-7595.md
Comment (xinbenlv) [inline ERCS/erc-7595.md:36]: A wrapper contract could do, I think

Comment (harpocrates555) [reply ERCS/erc-7595.md]: Wrapper contract is implemented for existing ERC-721 collections that would like to utilize the functionality of ERC-20 based collateral attached to the NFT collection without making too many changes to their NFT collection.

We also suggest a standard for ERC-721 creators who want additional customization, etc.

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md:65]: The proposed standard is focused on ERC-20 based collateral only. The proposed standard doesn't include ERC-721 as collateral to an ERC-721 collection.

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md]: Edited. NFTfi was referred as an emerging trend. We're NOT affiliated with the NFTfi project if such project exists.

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md]: All EIP updated for ERC.

**Comment by harpocrates555** [comment]
@xinbenlv , after changing all paths from ./eip-721.md to ./erc-721.md, we received the errors. Please advise.

**Comment by SamWilsn** [comment]
> after changing all paths from ./eip-721.md to ./erc-721.md, we received the errors. Please advise.

`eipw` is correct here. You need to use `./eip-xxxx.md` (and not `./erc-xxxx.md`) because of a quirk of our rendering system.

**Comment by harpocrates555** [comment]
@SamWilsn and @xinbenlv : everything updated. Thank you so much for your help/feedback. Looking forward to the next steps.

**Comment by SamWilsn** [comment]
I'll leave this to @xinbenlv to finish reviewing.

**Comment by harpocrates555** [comment]
Dear team, we would like to check in on the progress of EIP 7595. Please let us know if there are any actions required from our side.

@xinbenlv @SamWilsn @abcoathup

**Comment by Cosmos4k** [comment]
Dear team, we would like to check in on the progress of EIP 7595. Please let us know if there are any actions required from our side.

@xinbenlv @SamWilsn @abcoathup

File: ERCS/erc-7595.md
Comment (SamWilsn) [inline ERCS/erc-7595.md:16]: This is likely a me problem, but I'm not sure your abstract gives enough detail. Could you maybe add a sentence or two that describes the intended flow? Keep it brief, and try not to repeat your other sections.

---

What I'm getting from reading _just_ the abstract is like... anyone can deposit ether or ERC-20 tokens into the 721 contract (associated with a particular tokenId), and the owner of that token can withdraw the ether/ERC-20? Why?

File: ERCS/erc-7595.md
Comment (SamWilsn) [inline ERCS/erc-7595.md:20]: > NFTfi

This is not a common enough abbreviation to use without expanding it. Since you only use it once, perhaps:

```suggestion
The emerging trend of NFT finance focuses on the NFT floor price to enable the market value of the NFT serve as a collateral in lending protocols. The NFT floor price is susceptible to the supply-demand dynamics of the NFT market, characterized by higher volatility compared to the broader crypto market. Furthermore, potential price manipulation in specific NFT collections can artificially inflate NFT market prices, impacting the floor price considered by lending protocols. Relying solely on the NFT floor price based on market value is both unpredictable and unreliable.
```

File: ERCS/erc-7595.md
Comment (SamWilsn) [inline ERCS/erc-7595.md:201]: Is this name a placeholder, or is this the intended final name for the interface?

File: ERCS/erc-7595.md
Comment (SamWilsn) [inline ERCS/erc-7595.md:479]:
```suggestion
`blackHole` has been designed to prevent the transfer of any tokens from itself and can only perform read operations. It is intended to be used with the Envious extension in implementations related to commission harvesting.
```

use backticks for any inline code.

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md:201]: IEnviousHouse is the final name of the interface

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md:479]: backticks added

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md]: Added to the commit

File: ERCS/erc-7595.md
Comment (harpocrates555) [reply ERCS/erc-7595.md]: Added to the commit

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**
